engine/security/src/main/scala/pl/touk/nussknacker/ui/security/oauth2/CirceDurationConversions.scala

```diff
@@ -2,7 +2,7 @@ package pl.touk.nussknacker.ui.security.oauth2
 
 import io.circe._
 
-import scala.concurrent.duration.{Deadline, FiniteDuration, MILLISECONDS}
+import scala.concurrent.duration.{Deadline, FiniteDuration, SECONDS}
 
 protected[oauth2] trait CirceDurationConversions {
```

Comment:
I think this trait/codecs should be named e.g. OAuth codecs or EpochSecondsCodecs, because it's not obvious without looking at implementation how they work?